CMakeLists.txt

```diff
@@ -131,15 +125,13 @@ if (BUILD_STATIC)
         ${Boost_LIBRARIES}
         ${ODB_PATH}/libodb-sqlite.a
         ${ODB_PATH}/libodb.a
-        sqlite3
```

Comment:
Has the Docker build being tested after removing this? If it passes, then line 10 in `CMakeLists.txt` should be no longer necessary:

> set (CMAKE_C_FLAGS " -DSQLITE_ENABLE_UNLOCK_NOTIFY -DSQLITE_OMIT_LOAD_EXTENSION")